convert2rhel/main.py

```diff
@@ -363,3 +355,31 @@ def rollback_changes():
             loggerinst.info("During rollback there were no backups to restore")
         else:
             raise
+
+    return
+
+
+def print_info_after_rollback(pre_conversion_results, include_all_reports):
+    """Print after-rollback messages. Also determine if there is a report to print or
+    report should't be printed after failure in rollback."""
+    if backup.backup_control.rollback_failed:
+        loggerinst.critical_no_exit(
+            "Rollback of system wasn't completed successfully.\n"
+            "The system is left in an undetermined state that Convert2RHEL cannot fix.\n"
+            "It is strongly recommended to store the Convert2RHEL logs for later investigation, and restore"
+            " the system from a backup.\n"
+            "Following errors were captured during rollback:\n"
+            "%s" % "\n".join(backup.backup_control.rollback_failures)
+        )
+
+        return
+
+    elif not pre_conversion_results:
+        loggerinst.info("\nConversion interrupted before analysis of system completed. Report not generated.\n")
+
+    else:
+        report.summary(
+            results=pre_conversion_results,
+            include_all_reports=include_all_reports,
+            disable_colors=logger_module.should_disable_color_output(),
+        )
```

Comment:
Blocker. This is out of scope of the PR, please open a new one for this refactor and technical debt fixing :pray:

Reply:
Checked again and missed the `rollback_failed` check in the initial review, thus thinking this was just a refactor that was unrelated